src/core/public/chrome/chrome_service.tsx

```diff
@@ -71,6 +72,9 @@ export interface ChromeBrand {
 /** @public */
 export type ChromeBreadcrumb = EuiBreadcrumb;
 
+/** @public */
+export type HomePageBranding = Branding;
```

Comment:
I wonder if we should keep it inline with ChromeBranding?

Reply:
Changed the interface name to ChromeBranding.

ChromeBadge is being used in <HeaderBadge> component; for ChromeBrand interface, it is being used in setbrand() and getBrand() functions. The functions have no usage in the current application, but might be used in EUI or xPack. Decided to keep them as original.